hutch_python/mpl_config.py

```diff
@@ -1,2 +1,6 @@
-import matplotlib
-matplotlib.use('Qt5Agg')
+# noqa
```

Comment:
Does this work on a single line? Interesting...

Reply:
Argh, unfortunately it does not. That's what's causing the CI failures:
```
hutch_python/mpl_config.py:6:1: E402 module level import not at top of file
hutch_python/mpl_config.py:6:1: F401 'matplotlib.pyplot' imported but unused
```
Instead, I think, this should be added as an exception in `/.flake8`, perhaps:

```
[flake8]
exclude = .git,__pycache__,build,dist,versioneer.py,hutch_python/mpl_config.py
```